src/analysis/retail/priest/shadow/modules/spells/ShadowWordDeath.tsx

```diff
@@ -10,7 +12,10 @@ import { SHADOW_WORD_DEATH_EXECUTE_RANGE } from '../../constants';
 class ShadowWordDeath extends ExecuteHelper {
   static executeSources = SELECTED_PLAYER;
   static lowerThreshold = SHADOW_WORD_DEATH_EXECUTE_RANGE;
+  static singleExecuteEnablers: Spell[] = [SPELLS.DEATHSPEAKER_TALENT_BUFF];
+  //static executeOutsideRangeEnablers: Spell[] = [TALENTS.INESCAPABLE_TORMENT_TALENT]; //Need to fabricate a buff for when Inescapable Torment(mindbender) is active.
```

Comment:
we could modify `ExecuteHelper` a bit to make it support using other start/stop events. you wouldn't get the convenience of setting it with a prop like this, but you could add your own event listeners for mindbender and call `applyExecuteEnabler` and `removeExecuteEnabler` when it spawns/dies rather than fabricating a buff

Reply:
That would work great for this.

Reply:
I looked into this a little bit. Looks like either way you need fabrication because the mindbender doesn't have a death event https://www.warcraftlogs.com/reports/gPq7Kyv46RCmkzfn#fight=6&type=summary&pins=0%24Separate%24%23244F4B%24casts%240%240.0.0.Any%24214110952.0.0.Priest%24true%240.0.0.Any%24false%24200174&hostility=&source=30.1&view=events

so i'm not gonna worry about doing that as part of this PR. i can do that separately and then you can do the mindbender fabrication (either way) and then either use a buff or manually call apply/remove.

Reply:
Sounds good.  Its something I will work on in the future.